src/main/webapp/resources/js/views/BagDetailsWindow.js

```diff
@@ -84,7 +85,15 @@ Ext.define('BagDatabase.views.BagDetailsWindow', {
                 bagId: this.bagId,
                 data: []
             }]
-        }];
+        }, {
+            xtype: 'tagGrid',
+            region: 'east',
+            itemId: 'tags',
```

Comment:
Adding the `split: true` property here should add a bar between it and the center panel.